pages/clubs/Running.jsx

```diff
@@ -41,7 +44,7 @@ var Running = React.createClass({
     httpRequest.send();
   },
   componentDidMount: function() {
-    this.fetchJSON("https://mozilla.github.io/learning-networks/clubs/clubs-resources.json",this.setData);
+    this.fetchJSON(`${TEACH_API_URL}/api/clubsguides/categories?expand=clubsguides`,this.setData);
```

Comment:
why is this not just a `getClubsGuides` function in the teach-api.js library? That's what the library is for =)